tests/mypy/mypy_transient_function_wrapper_t1.py:56: error: Argument 2 to "transient_function_wrapper" has incompatible type "None"; expected "str"  [arg-type]
tests/mypy/mypy_transient_function_wrapper_t1.py:66: error: Argument 1 to "__call__" of "TransientDecorator" has incompatible type "Callable[[Callable[..., Any], tuple[Any, ...], dict[str, Any]], Any]"; expected "Callable[[Callable[[VarArg(Never), KwArg(Never)], Never], Any, tuple[Any, ...], dict[str, Any]], Any] | Callable[[Any, Callable[[VarArg(Never), KwArg(Never)], Never], Any, tuple[Any, ...], dict[str, Any]], Any] | Callable[[type[Any], Callable[[VarArg(Never), KwArg(Never)], Never], Any, tuple[Any, ...], dict[str, Any]], Any]"  [arg-type]
Found 2 errors in 1 file (checked 1 source file)
